context/discovery-context.jsonld

```diff
@@ -9,7 +9,17 @@
       "@id": "discovery:ThingLink",
       "@type": "@id"
     },
-    "registration": {
+	"ThingCollection": {
+      "@id": "discovery:ThingCollection",
+      "@type": "@id"
+    },
+	"total": "discovery:contains",
+	"members": [
+	  "@graph"
```

Comment:
This perhaps should be td:Thing included from the td context, since in the intent is to return TDs (formatted as TDs) and not general RDF graphs.